### ISSUE TITLE
Update for new macOS

### ISSUE THREAD

**Issue by exgphe** [KD-AIP/Hackintosh-T5810#1]
Hello. Do you have an update for macOS Ventura or macOS Monterey? I have the same hardwares as yours, but I keep getting kernel panics either using my EFI or modifying your EFI.
Thank you very much!

**Comment by KD-MM2** [comment]
Yes, I will but I can't promise that it will be soon. Maybe i will start to rework on this efi. There is nothing difficult. Just upgrade the opencore version, one by one version. Not jumping from present version to the lastest.

**Comment by exgphe** [comment]
I now doubt that would work. From my experience as well as other’s, T5810 could work fine in Big Sur, but once upgraded to Monterey, there would be an instant kernel panic after booting that cannot be fixed due to a bug in Dell’s BIOS. See https://www.insanelymac.com/forum/topic/323781-dell-precision-tower-5810-need-help-installing-sierra-high-sierra-mojave-catalina/?do=findComment&comment=2795787

**Comment by KD-MM2** [comment]
Which version of your BIOS are you in now? Have you try downgrade to older version? Not clearly remember but I was got a bunch of kernel panic until downgrade the bios.

**Comment by exgphe** [comment]
I tried A34 and then downgraded to A09. Same results.

**Comment by KD-MM2** [comment]
Ok. Got that. Maybe the rework start on next week. I don't sure for the result but will try.

**Comment by exgphe** [comment]
Wish you good luck!

**Comment by exgphe** [comment]
For information, if you are encountering a kernel panic "Non-monotonic time" while booting, a solution has been found recently. See https://www.insanelymac.com/forum/topic/323781-dell-precision-tower-5810-need-help-installing-sierra-high-sierra-mojave-catalina/?do=findComment&comment=2796722
